remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java

```diff
@@ -159,10 +159,10 @@ public void testCreateTags()
         then(authorityServiceMock).shouldHaveNoMoreInteractions();
         then(taggingServiceMock).should().createTags(StoreRef.STORE_REF_WORKSPACE_SPACESSTORE, tagNames);
         then(taggingServiceMock).shouldHaveNoMoreInteractions();
-        final List<Tag> expectedTags = createTagsWithNodeRefs(tagNames);
-        assertThat(actualCreatedTags)
-            .isNotNull()
-            .isEqualTo(expectedTags);
+        final List<Tag> expectedTags = createTagsWithNodeRefs(tagNames).stream()
+                                            .peek(tag -> tag.setCount(0))
```

Comment:
This test verifies a different method -> `.createTags(List<Tag> tags, Parameters parameters)`, and you was working on method `.addTags(String nodeId, List<Tag> tags)`. You should introduce new tests which will test right method and your fix.

Reply:
Interesting why this test didn't fail 🤔

Reply:
Reverted the changes in the original test cases. Added a new test case for the above scenario.